Sources/CvGlobals.cpp

```diff
@@ -467,11 +316,10 @@ cvInternalGlobals::cvInternalGlobals()
 	, m_iStoreExeSettingsCorporationInfo(0)
 	, m_iStoreExeSettingsBonusInfo(0)
 	, m_bSignsCleared(false)
+#define ADD_TO_CONSTRUCTOR(VAR) \
```

Comment:
I may need to add a second parameter to the macro so that I can give different starting values. But these values should always get over-ridden... so.... idk.